README.md

```diff
@@ -25,34 +25,53 @@ Docker images are the basis of containers. Images are read-only, while container
 
 ## How to use this image?
 
+## The Easy Way
+
+### Using docker-compose
+```bash
+docker-compose up -d
+```
+
+### Starting a Kudu console
+```bash
+docker run --rm -it --link <kudu_tserver_name_or_id>:kudu_tserver <generated image name or id> kudu-ts-cli -server_address=kudu_tserver status
+```
+
+## The Long Way
+
 ### Building this Docker image
 ```bash
-docker build -t name kudu .
+docker build -t kudu .
 ```
 
+##
 ### Starting the Kudu Master
 ```bash
-docker run -d  --name kudu-master \
-    -h kudu-master    -p 8051:8051 \
-    kudu kudu-master  -fs_wal_dir /data/kudu-master  --logtostderr  --use_hybrid_clock=false  --logtostderr  && \
-docker logs -f kudu-master
+docker run -d --name kudu-master \
+    -h kudu-master -p 8051:8051 kudu kudu-master \
+    -fs_wal_dir /data/kudu-master --logtostderr \
+    --use_hybrid_clock=false
 ```
 
 ### Starting the Kudu TabletServer
 ```bash
-docker run -d  --name kudu-tserver1 \
-    -h kudu-tserver1   -p 8050:8050 \
-    kudu kudu-tserver  -fs_wal_dir /data/kudu-tserver  -tserver_master_addrs kudu-master  --use_hybrid_clock=false  --logtostderr  && \
-docker logs -f kudu-tserver1
+docker run -d --name kudu-tserver1 \
+    -h kudu-tserver1 -p 8050:8050 kudu kudu-tserver \
+    -fs_wal_dir /data/kudu-tserver --logtostderr \
+    -tserver_master_addrs kudu-master --use_hybrid_clock=false
 ```
 
```

Comment:
Since the EntryPoint has been changed, it is necessary that the COMMAND argument to " docker run" became an argument to EntryPoint :
https://docs.docker.com/reference/run/#cmd-default-command-or-options